### ISSUE TITLE
Split Feature has duplicate name and creates a shortcut collision

### ISSUE THREAD

**Issue by techdude1996** [bstroebl/DigitizingTools#52]
The Split Feature command found in your plugin shares the same name as QGIS's Split Feature command. When creating a keyboard shortcut for either function, the xml file reads:

    <act shortcut="V" name="Split Features"/>

When the file is read in by QGIS, both commands get the 'V' key as their shortcut as they share the same name. So, when I press 'V', QGIS doesn't know which command to call and nothing happens.

When saving the file back out, it now reads:

    <act shortcut="V" name="Split Features"/>
    <act shortcut="V" name="Split Features"/>

Here's a screenshot of the shortcuts window with the plugin enabled:
![image](https://user-images.githubusercontent.com/25308291/73953583-9f390b00-48ce-11ea-99cf-39b75ff6ebab.png)

As you can see, QGIS can't distinguish the one from the other and treats them as the same.

An "easy" fix would be to prefix the commands in the plugin, i.e. "Digitizing - Split Features"

**Comment by oskarlin** [comment]
Yes this would be a very welcome fix since I have to change the Keyboard Shortcuts every time I start QGIS.

**Comment by mperwira** [comment]
Hi,

I'm new in using this plugin and I found that not only on the Split Feature, it also happens on the Merge Feature.
Since this plugin is very useful for my works, so for now I have to change the shortcuts every time start the QGIS.
